trax/layers/rnn.py

```diff
@@ -260,7 +260,10 @@ def SRU(n_units, activation=None):
   Args:
     n_units: output depth of the SRU layer.
     activation: Optional activation function.
-
+    rescale: To offset the problem of the gradient vanishing in the h_t as a result
+    of light recurrence and highway computation for deeper layers, a scaling correction
+    alpha is applied as follows: (1 + exp(highway_bias) * 2)**0.5
```

Comment:
Is there a paper that explains this formulation? If so, it'd be good to include a link to it. Thanks!

Reply:
@lukaszkaiser I have included the link and the section of the paper, related to this change.